docs/tutorials/basic/symbol.md

```diff
@@ -328,4 +498,20 @@ ex.forward()
 ex.outputs[0].asnumpy()
 ```
 
+## How Efficient Is the Symbolic API?
+
+In short, it is designed to be very efficient in both memory and runtime.
+The major reason for introducing the Symbolic API is to bring the efficient C++
+operations in powerful toolkits, such as CXXNet and Caffe, together with the
+flexible dynamic NDArray operations. To maximize runtime performance and memory
+utilization, all of the memory and computation resources are allocated
+statically during the bind operation.
+
+The coarse-grained operators are equivalent to CXXNet layers,
```

Comment:
people rarely know cxxnet. Consider removing mention of it and use caffe instead?